### ISSUE TITLE
Set accessibilityLabel in Tabs.

### ISSUE THREAD

**Issue by sytolk** [wix/react-native-navigation#2045]
I have TabBased App and I want to write E2E Testing  with Appium:

https://github.com/garthenweb/react-native-e2etest#accessibilitylabel-vs-testid

But now I cannot select in Appium test Tabs Button/TouchableOpacity ElementByAccessibilityId and next to click(). Can we have accessibilityLabel in JSON tabs model ?

**Comment by stephmilovic** [comment]
Can this be addressed? label passes to name/accessibilityId in iOS but android gets nothing through to content-desc/id. Until this is fixed, Appium cannot be used to test in Android with this package.

**Comment by guyca** [comment]
Hey guys, I'm closing the issue as this is resolved in v2. Sorry for the inconvenience.
